_data/projects/search/releases/7.0/7.0.0.Beta1.yml

```diff
@@ -2,8 +2,31 @@ date: 2023-09-05
 announcement_url: https://in.relation.to/2023/09/05/hibernate-search-7-0-0-Beta1
 summary: >-
   JDK 11 as a baseline,
-  switch to Jakarta EE, compatibility with ORM 6.3 and no `-orm6` artifacts, 
+  switch to Jakarta EE, compatibility with ORM 6.3 without `-orm6` artifacts, 
   upgrade to Lucene 9 in the Lucene backend,
   published Hibernate Search BOM,
   experimental compatibility with Amazon OpenSearch Serverless,
-  drop compatibility with EOL'd Elasticsearch versions 5.x - 6.x
+  dropped compatibility with EOL'd Elasticsearch versions 5.x - 6.x,
+  other bugfixes and improvements
+maven:
```

Comment:
It's always been supported but is rarely used, because it's only useful if the artifacts change within a given series. See `6.0.0.Alpha*` for examples.